src/components/Input/Input.component.tsx

```diff
@@ -86,9 +87,12 @@ export const Input = forwardRef<HTMLInputElement, InputProps>((props, ref) => {
 				<StyledInput
 					ref={ref}
 					placeholder={inputLabel}
-					onFocus={(event: React.ChangeEvent<HTMLInputElement>) =>
-						togglePlaceholderValue(event, 'focus')
-					}
+					onFocus={(event: React.ChangeEvent<HTMLInputElement>) => {
+						if(onInputFocus){
+							onInputFocus();
+						}
+						togglePlaceholderValue(event, 'focus');
+					}}
```

Comment:
Please fix the discussed issue in bootstrap migration PR for input component.

Reply:
Can fix discussed changes in https://github.com/Brightscout/mm-ui-library/pull/72